webapp/src/modules/translation/locales/en.json

```
@@ -1214,6 +1214,11 @@
     "cross_chain_tx": {
       "body": "The transaction can take up to <highlight>3 minutes</highlight> to be processed.<br></br>You can also track its progress in the links below:",
       "view_transaction": "View transaction"
+    },
+    "bid_placed_success": {
+      "title": "Congratulations",
+      "message": "Bid for <strong>{assetName}</strong> has been placed successfully",
```

Would you mind adding the article here?
```suggestion
      "message": "A bid for <strong>{assetName}</strong> has been placed successfully",
```